Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.21 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
```

Comment:
kserve is based on go1.21, I don't think a good idea move it back.
it will probably fail to build.

Reply:
updated